app/__init__.py

```diff
@@ -31,14 +39,15 @@ def generate_sample_results(limit):
 @lsns.route('/')
 class LogsSearch(Resource):
     @lsns.expect(search_parser)
-    def get(self):
+    def post(self):
         args = search_parser.parse_args()
 
-        log_filename = 'generated.log'
-        result_id = searchIn(log_filename)
+        #log_filename = 'generated.log'
+        #result_id = searchIn(log_filename)
+        run_script_with_args(args['contains'], args['limit'])
 
         response = {
-            'result_id': result_id
+            'msg': 'Script submited!'#'result_id': result_id
```

Comment:
Is it intentional to have no new line here?

Reply:
Done